spec/active_triples/relation_spec.rb

```diff
@@ -478,16 +494,19 @@ class WithTitle
       include_context 'with symbol property' do
         let(:parent_resource) { ActiveTriples::Resource.new }
       end
+      
+      context 'and persistence config' do
+        before do
+          reflections
+            .property :moomin, 
+                      predicate: RDF::URI('http://example.org/moomin'), 
+                      persist_to: ActiveTriples::RepositoryStrategy
+        end
 
-      it 'sets a value' do
-        expect { subject.set(:moomin) }
-          .to change { subject.to_a }.to contain_exactly(:moomin)
-      end
-
-      it 'sets mulitple values' do
-        values = [:moomin, :snork]
-        expect { subject.set(values) }
-          .to change { subject.to_a }.to contain_exactly(*values)
+        it 'returns values with persistence strategy set' do
+          expect(subject.set(RDF::Node.new).map(&:persistence_strategy))
+            .to contain_exactly(an_instance_of(ActiveTriples::RepositoryStrategy))
+        end
       end
     end
```

Comment:
@no-reply Is is correct that these tests were removed?  This may be a bad merge of a rebase conflict.